src/components/Filter.test.tsx

```diff
@@ -1,17 +1,14 @@
 import { render, screen } from "@testing-library/react";
-import nock from "nock";
 import FilterContextProvider from "../filter/FilterContext";
 import { mockAttributes } from "../test/mock-data/List.fixtures";
 import { withTheme } from "../test/utils";
 import Filter from "./Filter";
 
-describe("Filter", () => {
-  beforeEach(() => {
-    nock("http://localhost")
-      .get("/vsd/hior_attributes/?page=1&page_size=100000&format=json")
-      .reply(200, mockAttributes);
-  });
+jest.mock("../hooks/useFetchAttributes", () => ({
+  useFetchAttributes: () => ({ data: mockAttributes, isLoading: false }),
```

Comment:
Een nadeel van deze oplossing is dat je hier een implementatie detail (react-queries return object) aan het mocken bent. Als je zou overstappen naar iets anders als RQ voor fetchen dan moet je de testen ook aanpassen.
Gebruik van nock voorkomt dit juist doordat het http requests onderschept. Vervolgens maakt het niet meer uit hoe (welke library) je requests doet.

Reply:
Yeah, indeed, we couldn't get nock to work with `Fetch`ing the CSV's. It was a hard to debug issue and because I had spent too much time on it, I went for this solution. I'm open to suggestions, but currently I don't see a solution with nock.

Reply:
Als ik deze code aan de test toevoeg slaagt de test;

```
beforeEach(() => {
  nock("http://localhost:3000")
   .get("/static/data/attributes.csv")
   .reply(200, mockAttributes);
});
```

Reply:
I'll give it another shot then. But this seems exactly the same as what we were trying... 🤔

How are you running the tests (with make/docker or locally) and on what OS?

Reply:
Locally, when i've got some more time I could also look at docker.